src/ci/docker/host-x86_64/x86_64-gnu-llvm-10/Dockerfile

```diff
@@ -19,8 +19,10 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
   libssl-dev \
   pkg-config \
   zlib1g-dev \
-  xz-utils \
-  nodejs
+  xz-utils
+
+RUN curl -sL https://nodejs.org/dist/v14.4.0/node-v14.4.0-linux-x64.tar.xz | tar -xJ
+ENV PATH="/node-v14.4.0-linux-x64/bin:${PATH}"
```

Comment:
Is npm not available through apt?

Reply:
Apparently not based on the previous error.